app/javascript/components/report-editor/basic-report-info.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Form, Field } from 'react-final-form';
+import { FinalFormField } from '@manageiq/react-ui-components/dist/forms';
+import { cleanVirtualDom } from '../../miq-component/helpers';
+import { required } from '../../forms/validators';
+
+const BasicReportInfo = ({ name, title, url }) => {
+  // set Initial values if editing
+  const [initialValues] = useState({ name, title });
+  // deletes unused components from virtual DOM
+  useEffect(() => cleanVirtualDom());
+  return (
+    <Form initialValues={initialValues} onSubmit={() => {}}>
+      {() => (
+        <React.Fragment>
+          <Field
+            label={__('Menu Name')}
+            name="name"
+            id="name"
+            maxLength={40}
+            validate={required}
+            data={{
+              'data-miq_observe': `{"interval" : ".5", "url": "${url}"}`,
```

Comment:
`"${url}"` is just a string containing ${url}, no variable substitution will happen without ```

Reply:
(Also, can you describe the approach in the description please .. this looks like the react part deals with presentation only for now, and keeps using the old observe requests for some reason?)

Reply:
One more .. note https://github.com/ManageIQ/react-ui-components/pull/96#issuecomment-468747768 please.

Reply:
I tried to remove quotes around the url and it returns this error. 

<img width="712" alt="Snímek obrazovky 2019-03-10 v 16 16 45" src="https://user-images.githubusercontent.com/32618096/54087111-12996c00-4350-11e9-97e4-ccd3b58f53cb.png">

It expects this format. 
<img width="562" alt="Snímek obrazovky 2019-03-10 v 16 19 41" src="https://user-images.githubusercontent.com/32618096/54087152-8e93b400-4350-11e9-8964-25b8f7c94b48.png">

Reply:
Oh, you have the backticks around the whole value, I get it now :).

What about something like..

```js
'data-miq_observe': JSON.stringify({interval: 0.5, url: url}),
```